agreement/fuzzer/fuzzer_test.go

```diff
@@ -196,7 +196,7 @@ func (n *Fuzzer) initAccountsAndBalances(rootSeed []byte, onlineNodes []bool) er
 		if err != nil {
 			return err
 		}
-		n.accountAccessors[i*2+0] = rootAccess
+		n.accountAccessors[i] = rootAccess
```

Comment:
we used to store here both the rootkey + partkey, but then stopped using it for the part keys. As a result, we need only half of the size of the array.